docs/install_guide.adoc

```diff
@@ -0,0 +1,180 @@
+= User Installation Guide
+
+== Getting Started
+
+You will need an operational operator deployed OpenStack system
```

Comment:
perhaps avoid 'operational' next to operator and consider rephrasing

e.g.

"Before installing the Watcher operator you first need a functional OpenShift installation with the required openstack operators including the Telemetry operator"